src/libraries/System.Drawing.Common/tests/ImageTests.cs

```diff
@@ -199,5 +199,16 @@ public void GetEncoderParameterList_ReturnsExpected(ImageFormat format, Guid[] e
                     paramList.Param.Select(p => p.Encoder.Guid));
             }
         }
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework throws ExternalException")]
+        [ConditionalFact(Helpers.IsDrawingSupported)]
+        public void Save_InvalidDirecotry_ThrowsDirectoryNotFoundException()
```

Comment:
Nit: typo in the name. 
```suggestion
        public void Save_InvalidDirectory_ThrowsDirectoryNotFoundException()
```

Reply:
Thanks. I should have caught that.